.travis.yml

```diff
@@ -5,39 +5,21 @@ language: go
 matrix:
   include:
     - go: 1.4
-      env:
-        - KUBE_TEST_API_VERSIONS="v1"
-          KUBE_TEST_ETCD_PREFIXES="registry"
-    - go: 1.3
-      env:
-        - KUBE_TEST_API_VERSIONS="v1beta3"
-          KUBE_TEST_ETCD_PREFIXES="kubernetes.io/registry"
 
 install:
-  - go get golang.org/x/tools/cmd/cover
-  - go get github.com/mattn/goveralls
   - go get github.com/tools/godep
   - ./hack/travis/install-etcd.sh
+  - ./hack/build-go.sh
+
+script:
```

Comment:
This is strictly better than mine because mine has an implied "make" which is redundant.